.devcontainer/devcontainer.json

```diff
@@ -39,7 +39,8 @@
 				"ms-python.isort",
 				"charliermarsh.ruff@prerelease",
 				"wayou.vscode-todo-highlight",
-				"ms-iot.vscode-ros@prerelease"
+				"ms-iot.vscode-ros@prerelease",
+				"ms-azuretools.vscode-docker"

```

Comment:
Why was this added

Reply:
That extension helps when editing dockerfiles / it adds support for the docker language

Reply:
Are we editing them in the devcontainer itself though?

Reply:
Yes, I was, and then vscode recommended me to install the extension, so I did lol